integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/cloud/AzureCloudProvider.java

```diff
@@ -111,24 +105,31 @@ public String region() {
     }
 
     @Override
-    public StackAuthenticationRequest stackauth() {
-        StackAuthenticationRequest stackauth = new StackAuthenticationRequest();
+    public StackAuthenticationV4Request stackauth() {
+        StackAuthenticationV4Request stackauth = new StackAuthenticationV4Request();
 
         stackauth.setPublicKey(getTestParameter().get(INTEGRATIONTEST_PUBLIC_KEY_FILE).substring(BEGIN_INDEX));
         return stackauth;
     }
 
     @Override
-    public TemplateV2Request template() {
-        TemplateV2Request t = new TemplateV2Request();
-
+    public InstanceTemplateV4Request template() {
+        InstanceTemplateV4Request t = new InstanceTemplateV4Request();
```

Comment:
please add a meaningful variable name

Reply:
you're absolutely right, but changing variable names are out of scope now. when a targeted refactor will happen, that will contain renaming stuffs.
tha main goal is just to make the code compilable :D

Reply:
i know it's not a big deal, i mean not blocker, but if we changing the code, then we should make these kind of easy renames like this, because later we will forget it for sure